cli/src/commands/init.rs

```diff
@@ -112,6 +113,17 @@ pub struct Opts {
 
 impl Opts {
 	pub async fn execute(&self, term: &Term) -> Result<()> {
+		// Remove legacy `.rivet` dir if exists
+		let legacy_project_meta_path = paths::project_root()?.join(".rivet");
+		if fs::metadata(&legacy_project_meta_path).await.is_ok() {
+			term::status::warn(
+				"Deleting legacy project metadata",
+				".rivet/ folder is moved to a global config",
+			);
+			fs::remove_dir_all(&legacy_project_meta_path).await?;
```

Comment:
Is there any migration to the global config planned for this? Or is a new `rivet init` expected on all projects?

Reply:
Nope. Not worth the time imo since it's not _really_ breaking, just annoying.

Reply:
Sounds good 👍